### PR TITLE
[nodejs][03.welcome-users] Corrected sample

### PR DESCRIPTION
**Pull request by joseques**
## Proposed Changes

- Replace `.bot` file names in `index.js` to the correct one.
- Rename `.bot` file to match sample name
- Match casing in README.md with bot filename
- Change the section `Deploy this bot to Azure` linking existing documentation
- Change `ConversationState` to `UserState` in `index.js` since this sample has the scope of `UserState`

### CODE REVIEW

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]
For [Deploy this bot to Azure](https://github.com/southworkscom/BotBuilder-Samples/blob/088536f34efba724bca8f8d70b25f9d8d01ea267/samples/javascript_nodejs/03.welcome-users/README.MD#deploy-this-bot-to-azure), see if  you can replicate the way it is explained [here](https://github.com/southworkscom/BotBuilder-Samples/blob/00d5391cd660b93c35cd9ff1c0211463e7d7b29d/samples/csharp_dotnetcore/17.multilingual-bot/readme.md#deploy-this-bot-to-azure).

**Comment by joseques** [comment]
Updated @VictorGrycuk

**Review by VictorGrycuk — APPROVED** [review]

File: samples/javascript_nodejs/03.welcome-users/README.MD [hunk 1]
```diff
@@ -38,15 +38,7 @@ Bots that are added directly by a user, are mostly personal (1:1) conversation b
...
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. 
-
-To install all Bot Builder tools - 
-
-Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher
-
-```bash
-npm i -g msbot chatdown ludown qnamaker luis-apis botdispatch luisgen
-```
+You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. In order to install this and other tools, you can read [Installing CLI Tools](../../../Installing_CLI_tools.md).
```
Comment (VictorGrycuk) [inline samples/javascript_nodejs/03.welcome-users/README.MD:41]: Please see if you can update this section to the latest one used in [netcore](https://github.com/southworkscom/BotBuilder-Samples/blob/78a021e12898d684f8c48bb72c6c2ca332e62c99/samples/csharp_dotnetcore/13.basic-bot/README.md#deploy-this-bot-to-azure).

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]
For [line 12](https://github.com/southworkscom/BotBuilder-Samples/pull/88/files#diff-8318bbe5906d2e8faa3e10876f459391R12), use `code formating` for the path.

**Review by VictorGrycuk — APPROVED** [review]

**Comment by joseques** [comment]
@VictorGrycuk Updated with changes in [line 12](https://github.com/southworkscom/BotBuilder-Samples/pull/88/files#diff-8318bbe5906d2e8faa3e10876f459391R12)

**Review by dfavretto — CHANGES_REQUESTED** [review]
Fix typo in [line 97](https://github.com/southworkscom/BotBuilder-Samples/blob/d04a523fe3f1dc6504d23016c5b4882bbe6a1d88/samples/javascript_nodejs/03.welcome-users/bot.js#L97) file `bot.js`

**Comment by joseques** [comment]
Updated @dfavretto

**Review by dfavretto — APPROVED** [review]

**Comment by dfavretto** [comment]
Closed due to Microsoft#719

**Comment by joseques** [comment]
Reopened for needed changes. I think the path pointing to an incorrect folder it's a needed change. @VictorGrycuk @dfavretto

**Comment by dfavretto** [comment]
Closed due to Microsoft#719

**Pull request closed without merging**